src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -66,12 +67,44 @@ public ProjectsWrapper queryCristinProjectsIntoWrapperObjectWithAdditionalMetada
         projectsWrapper.setProcessingTime(calculateProcessingTime(startRequestTime, endRequestTime));
         // TODO: NP-2385: Use Link header / Pagination data from Cristin response in the next two values
         projectsWrapper.setFirstRecord(0);
-        projectsWrapper.setNextResults(null);
+        projectsWrapper.setNextResults("");
```

Comment:
Is this really a case where we want an empty string? Here, I say we don't.

Reply:
It will come a value here in a later issue, when we have implemented logic for reading headers from backend call. Do you want me to use null for now until implementation?

Reply:
Fair enough, I won't be pedantic :D